src/functions/getEvlTestCertificates.ts

```diff
@@ -0,0 +1,13 @@
+import {EvlTestCertificatesDAO} from "../models/EvlTestCertificatesDAO";
+import {EvlTestCertificatesService} from "../services/EvlTestCertificatesService";
+import { HTTPResponse } from "../models/HTTPResponse";
+
+export const getEvlTestCertificates = () => {
+  const evlTestCertificatesDAO = new EvlTestCertificatesDAO();
+  const evlTestCertificatesService = new EvlTestCertificatesService(evlTestCertificatesDAO);
+
+  return evlTestCertificatesService.getEvlTestCertificates().then((data) => {
+      return new HTTPResponse(200, data);
+    });
```

Comment:
maybe add a catch block to catch any unwanted errors:
.catch((error: any) => {
     return new HTTPResponse(error.statusCode, error.body);
);

Reply:
> You should also implement some tests for validating the AC's. We are now using jest-cucumber for this. Please see branch feature/CVSB-7968 from tech-records and look in /tests/features
> 
> https://github.com/dvsa/cvs-svc-technical-records/tree/feature/CVSB-7968/tests/features

The reason for not writing the integration tests is the fact that this is a stubbed API and has no data source. The data is coming from a hardcoded json file. When the datasource is determined then we can do an iteration on this code and write tests for ACs.

Reply:
> maybe add a catch block to catch any unwanted errors:
> .catch((error: any) => {
> return new HTTPResponse(error.statusCode, error.body);
> );

catch block added